pkgs/servers/ankisyncd/default.nix

```diff
@@ -11,7 +10,8 @@ python3.pkgs.buildPythonApplication rec {
     owner = "ankicommunity";
     repo = "anki-sync-server";
     rev = version;
-    sha256 = "196xhd6vzp1ncr3ahz0bv0gp1ap2s37j8v48dwmvaywzayakqdab";
+    hash = "sha256-RXrdJGJ+HMSpDGQBuVPPqsh3+uwAgE6f7ZJ0yFRMI8I=";
+    fetchSubmodules = true;
```

Comment:
2.4.0 no longer requires submodules https://github.com/ankicommunity/anki-sync-server/tree/v2.4.0

Reply:
Thanks for the review!

anki-sync-server 2.4.0 still requires anki from pypi, so it's not going to be a straightforward change (and the update isn't as simple as bumping the version number)

The current discussion seems to favor just getting rid of this flavor of ankisyncd in favor of the rust one (being discussed in https://github.com/NixOS/nixpkgs/pull/224366 ) so I'd say to just not touch ankisyncd here more than we have to for now.

Reply:
Yup, as discussed in  #224366 and previously in this pr, I don't think it's worth updating it (nor is it easy to do so).
I'm just trying to leave it functioning at all since this PR would otherwise break it, and then we can deal with it separately